### PR TITLE
Class 01

### PR DESCRIPTION
**Pull request by Swaintek**
The assignment took about 4 hours.  Mostly struggles with the DOM manipulation, everything else was fairly easy.  Index.js is not an active file, just a placeholder for notes.  I did not need to use a prototype to HTML() and everything still seems to work.

### CODE REVIEW

File: articles.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+var articleConstructor = function (data, num) {
+  $('article:first').removeAttr('class').removeAttr('id').clone().appendTo('main');
+  $('article:last').addClass(data.category).attr('id', num);
+  $('article#'+num+' .title').text(data.title);
+  $('article#'+num+' .author').text(data.author);
+  $('article#'+num+' .body').html(data.body);
+  $('article#'+num+' .age').text(Math.floor(
+    (new Date - new Date(data.publishedOn))/86400000) + ' Days Old'
+  );
+};
```
Comment (joelstanner) [inline articles.js:10]: Great. I love the addition of a unique number ID.

File: scripts/index.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+var Article = function (props) {
+  this.author = props.authoer;
+  this.whatever = props.whatever;
+  articles.push(this);
+};
+
+Article.prototype.toHTML = function() {
+  var $clonedARticle = $('article#template').clone();
+  $clonedArticle.removeAttr('id');
+  $clonedArticle.find('h1.author').html('<a href "' + articles[i].authorURL + '" ' + articles[i].author);
```
Comment (joelstanner) [inline scripts/index.js:22]: This is a bit awkward to be inserting html from here. What about using  `$clonedArticle.find('...').attr('href', this.authorUrl);`

Comment (Swaintek) [inline scripts/index.js:22]: My index.js file isn't active, I'm just using it for notes in class.

Comment (joelstanner) [inline scripts/index.js:22]: Gotcha. Just to keep your master branch clean, you may want to not commit this to a pull request to master if it's not being used unless there is good reason to include it.

**Pull request merged**